com.endlessm.EknServices.json

```diff
@@ -17,8 +17,8 @@
             "sources": [
                 {
                     "type": "git",
-                    "url": "https://c5a7fd3e12c1d7b20bbff48a541d36f92d1e5a2d@github.com/endlessm/eos-knowledge-services.git",
-                    "branch": "master"
+                    "path": ".",
+                    "branch": "@GIT_CLONE_BRANCH@"
```

Comment:
Unfortunately I don't think this will work if you don't make the manifest a template.
eos-knowledge-services uses `jenkins-build-flatpak-manifest` to be built, which will only substitute `GIT_CLONE_BRANCH` in if the file name ends with `.in`: https://github.com/endlessm/eos-build/blob/master/ci-integration/jenkins-build-flatpak-manifest#L162

Reply:
Oh, good catch.